database/knex.js

```diff
@@ -1,11 +1,20 @@
 const expect = require("expect-runtime");
 const connection = process.env.DATABASE_URL
 expect(connection).to.match(/^postgresql:\//);
+const urlregexp = /postgresql:\/\/(.+):(.+)@(.+):(\d+)\/(.+)\?ssl=true/g;
```

Comment:
When I upgrade the node version (to get es6 syntax support in the script), I get a connection timeout from knex/pg. This is a work around created a while ago that is still to be merged from a while ago. Introducing it as part of this script.